tensorboard/webapp/routes/dashboard_deeplink_provider_types.ts

```diff
@@ -27,3 +27,5 @@ export const PINNED_CARDS_KEY = 'pinnedCards';
 export const RUN_COLOR_GROUP_KEY = 'runColorGroup';
 
 export const TAG_FILTER_KEY = 'tagFilter';
+
+export const RUN_REGEX_FILTER_KEY = 'runRegexFilter';
```

Comment:
Perhaps simpler, `runFilter` to match `tagFilter`? They are both regexes.

Reply:
Isn't `tagFilter ` related to another filter? (where the arrow points at)
![Screen Shot 2021-11-10 at 10 53 29 AM](https://user-images.githubusercontent.com/1131010/141175315-fa5e339e-f02d-4462-92e9-8c44162ae6e3.png)
t)

Reply:
So, in L29, we have "tagFilter" instead of "tagRegexFilter" even though tag filter also let you specify a regex.

Likewise, I am asking  if we can drop word" regex" from "runRegexFilter" like we did for tag one.

Reply:
ah I see. I thought you want to rename `runRegexFilter` to `tag..`